patches/@lerna+version+5.6.2.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/node_modules/@lerna/version/lib/git-add.js b/node_modules/@lerna/version/lib/git-add.js
+index d24f8ca..7cd7c6d 100644
+--- a/node_modules/@lerna/version/lib/git-add.js
++++ b/node_modules/@lerna/version/lib/git-add.js
+@@ -29,8 +29,8 @@ function maybeFormatFile(filePath) {
+   if (!prettier) {
+     return;
+   }
+-  const config = resolvedPrettier.resolveConfig.sync(filePath);
+   try {
++    const config = resolvedPrettier.resolveConfig.sync(filePath);
```

Comment:
***IF APPROPRIATE*** (I have no idea), please send a PR upstream to lerna with this change, and track their response somewhere like https://github.com/endojs/endo/issues/576 . But probably not there since it is not endo relevant. Or is it?

Reply:
Here's what we did in Endo https://github.com/endojs/endo/blob/master/patches/%40lerna%2Bversion%2B5.6.2.patch

Reply:
I don't understand the actual purpose of either patch so I don't know:
   * Are they trying to address a similar purpose?
   * Does either one subsumes the other? 
   * Is either is something we'd want lerna to adopt upstream? 
   * Should either or both be added to https://github.com/endojs/endo/issues/576 ?

Reply:
> Are they trying to address a similar purpose?

Yes. We're on an old version of Lerna that's not compatible with the latest version of Prettier. Both patches make Lerna able to use the latest Prettier. 

> Does either one subsumes the other?
The one in Endo is a little cleaner way to go about it.

> Is either is something we'd want lerna to adopt upstream?

Their latest releases don't need it. Their old release I don't expect they'll mess with.

> Should either or both be added to https://github.com/endojs/endo/issues/576 ?

Nope.

Reply:
It's a patch for an old Lerna 5 (upstream is on 7) triggered by a breaking change in Prettier.  No relevance to Endo, except that @turadg pointed out there's a different/better patch for it in the Endo monorepo.  I'll use that one in the current PR instead.

Reply:
Thanks all! Resolving.